features/unit/v2indexer_rekey_paths.feature

```diff
@@ -0,0 +1,33 @@
+@unit
```

Comment:
could you put this in `v2indexerclient_paths.feature`? You can tag each scenario with `unit.indexer.rekey`.

Reply:
Ah, I had syntax errors from improperly trying to tag just the scenarios, thought it was impossible - looks like I was just doing it wrong :+1: